packages/sw-background-sync-queue/src/lib/queue.js

```diff
@@ -0,0 +1,143 @@
+import stringHash from 'string-hash';
+import idbQHelper from './idbHelper';
+import requestManager from './requestManager';
+import bcmanager from './broadcastManager';
+
+let _queue;
+let _config;
+
+/**
+ * Core queue class that handles all the enqueue and dequeue
+ * as well as cleanup code for the background sync queue
+ * @class Queue
+ */
+class Queue {
+	/**
+	 * Creates an instance of Queue.
+	 *
+	 * @memberOf Queue
+	 */
+	constructor() {
+		this.initialize();
```

Comment:
done, but a question.
I am not declaring a function async. its just i am calling another async function. not awaiting on it.
still a problem?